Formula/d/deindent.rb

```diff
@@ -0,0 +1,42 @@
+class Deindent < Formula
+  desc "Command-line utility to format overly-indented text"
+  homepage "https://github.com/adriangoransson/deindent"
+  url "https://github.com/adriangoransson/deindent/releases/v1.0.1"
+  sha256 "ce10f06922ddac95167beb1476bd83016e02952142876f6fec5dd8ae3c5de965"
```

Comment:
It seems that the checksum is incorrect, please see the ci result.

Reply:
Thanks for reviewing! I had to drop this a little suddenly.

I wondered why I couldn't get it to build locally so I hoped it was something wrong with my env. Noticed now that I had the wrong `url`! I'll go ahead and draft this now that I can debug it locally and then publish it again! 👍

Reply:
Ok, I got it running now. But turns out the audit fails as the repo is too unknown. Didn't realize this was a requirement. So feel free to close if necessary!

Reply:
Yeap, this is really new repo indeed and so close this PR in this time. You can create a custom tap to maintain your own tap, see [documentation](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap#maintaining-a-tap).